File: examples/kotlin/ftl-module-time/src/main/kotlin/ftl/time/Time.kt
package ftl.time

import ftl.builtin.Empty
import xyz.block.ftl.Context
import xyz.block.ftl.Verb
import java.time.OffsetDateTime

// Make time an OffsetDateTime once schema extraction is fixed
data class TimeResponse(val time: String)

@Verb
fun time(context: Context, req: Empty): TimeResponse {
  return TimeResponse(time = OffsetDateTime.now().toString())
}

File: go-runtime/ftl/testdata/go/time/time.go
package time

import (
	"context"
	"time"
)

type TimeRequest struct{}
type TimeResponse struct {
	Time time.Time
}

// Time returns the current time.
//
//ftl:verb export
func Time(ctx context.Context, req TimeRequest) (TimeResponse, error) {
	return TimeResponse{Time: time.Now()}, nil
}
